python/ray/autoscaler/v2/tests/test_node_provider.py

```diff
@@ -610,6 +610,112 @@ def test_inconsistent_pods_raycr_scale_down(self):
             },
         ]
 
+    def test_scale_down_multiple_pods_of_node_type(self):
+        """
```

Comment:
Is this relevant to this PR?

Reply:
> I left the other test case (test_scale_down_multiple_pods_of_node_type) in this PR since there isn't currently a test case that covers terminating multiple nodes at once.

just saw your comment here